### PR TITLE
Adding kgu05's coordinators

### LINKED ISSUE #3966: Hello!
Hello! :wave: Please add me as a collaborator to this repo.

### PR DESCRIPTION
**Pull request by kgu05**
@githubteacher
this closes #3966

### CODE REVIEW

**Comment by bingsu** [comment]
@kgu05 This looks great 🎉

File: kgu05.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: kgu05
+latitude: 38.850108
+longitude: -77.039175
+---
```
Comment (hectorsector) [inline kgu05.json]: @kgu05 
Great work so far! Your pin is in the root directory, though. Just make sure you move it into the `pins` folder, and that should allow the automated tests to pass!

**Comment by brianamarie** [comment]
Hi @kgu05 I'm going to go ahead and make the changes now.

**Comment by brianamarie** [comment]
@kgu05 I'm going to merge now. Nice work!

**Pull request merged**